app/models/ticket_request.rb

```diff
@@ -354,27 +353,28 @@ def build_ticket_request_event_addons_from_params(build_params)
   end
 
   def active_addons
-    ticket_request_event_addons.where('quantity > ?', 0)
+    @active_addons ||= ticket_request_event_addons.where('quantity > ?', 0)
```

Comment:
Addons can change during an event. Also before a payment is made final. 
Need to ensure these memoizations catch changes.

Reply:
Reviewed and tested. Looks good.